demo-shell/src/app/components/form/form.component.scss

```diff
@@ -32,7 +32,10 @@
 }
 
 .app-form-config-editor {
-    height: 500px !important;
+    height: 500px;
+    #adf-form-config-editor {
```

Comment:
can you put empty line before that selector?

Reply:
added

Reply:
I can't see this change, did you push it?

Reply:
pushed